proto/tunnel/tunnel.pb.go

```diff
@@ -782,19 +796,207 @@ func file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_init() {
 	out := protoimpl.TypeBuilder{
 		File: protoimpl.DescBuilder{
 			GoPackagePath: reflect.TypeOf(x{}).PkgPath(),
-			RawDescriptor: file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_rawDesc,
+			RawDescriptor: file_proto_tunnel_tunnel_proto_rawDesc,
 			NumEnums:      3,
 			NumMessages:   5,
 			NumExtensions: 0,
 			NumServices:   1,
 		},
-		GoTypes:           file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_goTypes,
-		DependencyIndexes: file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_depIdxs,
-		EnumInfos:         file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_enumTypes,
-		MessageInfos:      file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_msgTypes,
+		GoTypes:           file_proto_tunnel_tunnel_proto_goTypes,
+		DependencyIndexes: file_proto_tunnel_tunnel_proto_depIdxs,
+		EnumInfos:         file_proto_tunnel_tunnel_proto_enumTypes,
+		MessageInfos:      file_proto_tunnel_tunnel_proto_msgTypes,
 	}.Build()
-	File_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto = out.File
-	file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_rawDesc = nil
-	file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_goTypes = nil
-	file_github_com_openconfig_grpctunnel_proto_tunnel_tunnel_proto_depIdxs = nil
+	File_proto_tunnel_tunnel_proto = out.File
+	file_proto_tunnel_tunnel_proto_rawDesc = nil
+	file_proto_tunnel_tunnel_proto_goTypes = nil
+	file_proto_tunnel_tunnel_proto_depIdxs = nil
+}
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ context.Context
+var _ grpc.ClientConnInterface
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the grpc package it is being compiled against.
+const _ = grpc.SupportPackageIsVersion6
```

Comment:
Not sure if this was intentional or not, but this is downgrade of the method of compiling the proto code to Go. The new method uses a separate protoc plugin for generating the grpc service: google.golang.org/grpc/cmd/protoc-gen-go-grpc. You should end up with a separate _grpc.pb.go file.

Reply:
thanks for catching this. it was generated by compile_proto.sh, which seems to be using `--go_out=plugins=grpc,`.

will update the script shortly.